line_obfuscator.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"fmt"
+	"go/ast"
+	"golang.org/x/tools/go/ast/astutil"
+	mathrand "math/rand"
+)
+
+// PosMax is the largest line or column value that can be represented without loss.
+// Incoming values (arguments) larger than PosMax will be set to PosMax.
+// Source: https://golang.org/src/cmd/compile/internal/syntax/pos.go
+const PosMax = 1 << 30
+const PosMin = 1
+
+var emptyLine = &ast.CommentGroup{List: []*ast.Comment{{Text: "//line :1"}}}
```

Comment:
this is only used in line 31 so don't make a extra variable and just use it directly